ocaml/xapi/nm.ml

```diff
@@ -856,7 +856,7 @@ let bring_pif_down ~__context ?(force = false) (pif : API.ref_PIF) =
                        !Xapi_globs.firewall_port_config_script
                        ["close"; "4789"; "udp"]
                 )
-            | `gre ->
+            | `gre | `vxlan_mesh ->
```

Comment:
UDP port 4789 is opened for `\`vxlan_mesh` when `bring_pif_up`, why not close it when not used?

Reply:
> If last tunnel with VxLAN protocol is unplugged, close VxLAN UDP port.

Ok, we shoudl close the port if it's last tunell with VxLAN protocol.